lib/replace/wscript

```diff
@@ -362,12 +362,9 @@ removeea setea
     conf.env.intl_libs=''
     if not Options.options.disable_gettext:
         # any extra path given to look at?
-        if not Options.options.gettext_location == 'None':
-           conf.env['CFLAGS'].extend(["-I%s" % Options.options.gettext_location]);
```

Comment:
Wouldn't conf.ADD_EXTRA_INCLUDES be more suitable? No a waf expert but it always seems more reasonable to me to separate compiler translation flags from pre-processor flags.
If that's not suitable for some reason, I see we also have ADD_CFLAGS...